src/test/ui/or-patterns/multiple-pattern-typo.stderr

```diff
@@ -0,0 +1,52 @@
+error: unexpected token `||` after pattern
+  --> $DIR/multiple-pattern-typo.rs:8:15
+   |
+LL |         1 | 2 || 3 => (),
+   |               ^^ help: use a single `|` to specify multiple patterns: `|`
```

Comment:
`to separate multiple alternative patterns` maybe?

Reply:
Much better. 👍